indexer/services/l2/bridge/standard_bridge.go

```diff
@@ -76,11 +80,7 @@ func (s *StandardBridge) GetWithdrawalsByBlockRange(ctx context.Context, start,
 				}
 			}
 
-			hash, err := withdrawals.WithdrawalHash(withdrawalInitiated)
-			if err != nil {
-				return nil, err
-			}
-
+			hash := common.Hash(withdrawalInitiated.WithdrawalHash)
```

Comment:
This is also available in the event. Rederiving it is cool but the extra deps for it isn't worth it

Reply:
You could hash the withdrawal locally and compare it against the event's hash to be sure that you are using software that supports the correct version of withdrawal serialization but maybe its not useful for generalized indexing, this is more useful for software that builds proofs

Reply:
Yeah i agree the integrity check is good. I'll probably utilize this in the new indexer code pathway. This change is reverted now that waits.go is in op-service